### DISCUSSION TITLE
Make AXFloatingWindow floating/always on top?

### DISCUSSION THREAD

**Discussion by NathanR-85** [asmvik/yabai#2290] [Q&A]
I'm trying to ensure that an AXFloatingWindow in a certain application remains visible and on top, even when switching to other applications. 

This window has a peculiar behavior where it cannot become the active window even when selected, which is fine but might be an important piece of information. 

Also, since there are multiple AXFloatingWindows in the app, I'd like to target one with a particular Window Title (e.g. contains the word "Video").

Does anyone know if this is possible with yabai and how to achieve it?

Even if we have to change the subrole to AXStandardWindow, or something weird like that?

Thanks so much in advance!

**Comment by NathanR-85** [comment]
Anyone?

**Comment by Bellavene** [comment]
?

```
yabai -m rule --add app="^Application*$" \
         role="AXLayoutArea" \
         subrole="AXFloatingWindow" \
         title='My beloved title' \
         manage=off \
         sub-layer=above \
         sticky=on \
         display=^1 \
         grid=1:1:0:0:1:1
```

Also, this script will cycle all between available windows

```
yabai -m query --spaces | jq -re '.[] | select(."is-visible"== true).index' | xargs -I{} yabai -m query --windows --space {} | jq -sre 'add | sort_by(.display, .frame.x, .frame.y, .id) | nth(index(map(select(."has-focus" == true))) - 1).id' | xargs -I{} yabai -m window --focus {}
```
